### PR TITLE
feature: create layout for the website

### PR DESCRIPTION
**Pull request by stephanieesenwa**
#### What does this PR do?
This PR create the layout structure for the app

#### Description of Task to be completed?
- create the css grid layout
- add the different HTML sections

#### How should this be manually tested?
- fetch the branch manually
- run npm install
- run npm watch:sass
- view on browser

#### What are the relevant pivotal tracker stories?
 #12345

#### Questions:
N/A

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.body {
+  display: grid;
+  grid-template-rows: 1fr 75vh 1fr; }
```
Comment (onwuvic) [inline css/style.css:3]: @stephanieesenwa bring the curly brace down to a new line

File: index.html [hunk 2]
```diff
@@ -8,8 +8,24 @@
     <link rel="stylesheet" href="./css/style.css">
 </head>
 <body>
-    <div>
-        The Layout
+    <div class="body">
```
Comment (onwuvic) [inline index.html:11]: @stephanieesenwa rename the body class to container

File: css/style.css (hunk 1 above)
Comment (stephanieesenwa) [reply css/style.css:3]: @onwuvic  oh that is the way the compile version is.

Comment (onwuvic) [reply css/style.css:3]: @stephanieesenwa I have seen it now. Great work 👍🏽

**Comment by onwuvic** [comment]
@stephanieesenwa  This looks good to me. You can merge it now.

**Pull request merged**